FirebaseAuth/Tests/Sample/Sample/MainViewController.m

```diff
@@ -252,6 +253,8 @@ - (void)updateTable {
       [weakSelf oobSection],
       // Auto Tests
       [weakSelf autoTestsSection],
+      // Passkey
```

Comment:
let's move this section above the Auto tests, which is actually deprecated.

Reply:
Updated